ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -869,7 +869,9 @@ long getPartitionCount() {
     long getPartitionReadWriteCount() {
       long count = 0;
       for (AmbryPartition partition : partitionMap.values()) {
-        if (partition.getPartitionState() == PartitionState.READ_WRITE) {
+        // TODO note that currently even partially read write partitions are treated as read write.
```

Comment:
nit: I have a habit that when I implement a feature, I use FEATURE_NAME_TODO as the comment like EFFICIENT_META_TODO
I guess we may have many TODO in the existing code. It's easy to lose track of the generic TODO.

Reply:
Thats a great idea. It will also make my job easier finding all the left out todos to close the project. I will update all the relevant TODOs in the next PR.